src/ophyd_async/core/_hdf_dataset.py

```diff
@@ -19,6 +19,10 @@ class HDFDataset:
     dtype_numpy: str = ""
     multiplier: int = 1
     swmr: bool = False
+    # Represents explicit chunk size written to disk.
+    # The first dimension represents the number of frames/points
+    # per equally-sized chunk in the first chunking dimension.
```

Comment:
I don't think the second half of this is actually always true, we happen to make this so for areaDetector, but in the general case the chunk size is just that, the size of the HDF chunk. This may or may not correspond to a multiple of images, it might be half an image in one direction and 15 in the other, but its magnitude is in the base datatype of the dataset.

I wonder if `chunk_shape` might be a better name to make people think of `shape` in the output of `describe()`?

Reply:
I actually had the same thought yesterday about calling it `chunk_shape`. If it's not too late, I'd +1 to that.

Reply:
No preference from me about `chunk_shape` over `chunk_size` - perhaps lets discuss tomorrow at the sprint catchup, and we can settle on a naming scheme